Source/websocket/JSONRPCLink.h

```diff
@@ -583,13 +583,19 @@ namespace JSONRPC {
             , _localSpace()
             , _pendingQueue()
             , _scheduledTime(0)
+            , _versionstring()
         {
             if (localCallsign == nullptr) {
 	        static uint32_t sequence;
                 _localSpace = string("temporary") + Core::NumberType<uint32_t>(Core::InterlockedIncrement(sequence)).Text();
             } else {
                 _localSpace = localCallsign;
             }
+
+            uint8_t version = Core::JSONRPC::Message::Version(callsign + '.');
```

Comment:
Really needed the addition of the '.' to the callsign ?

Reply:
Yes, otherwise the Version method does not work correctly (and I did not want to change that)